### ISSUE TITLE
Don't let player capture Home and End keys

### ISSUE THREAD

**Issue by opusforlife2** [TeamPiped/Piped#3706]
### Official Instance

- [X] The bug is reproducible on the [official hosted instance](http://piped.video/), or is API-related.

### Describe the bug

Along with Pg Up and Pg Dn, the Home and End keys can be used to jump scroll pages. However, on Piped, the latter 2 keys are captured by the player, and instead of jumping to the page top and bottom, they jump to the video beginning and end.

### To Reproduce

1. Play any video.
2. Use the Pg Up and Pg Dn keys. These work fine.
3. Use the Home and End keys. These restart/end the video instead of scrolling the page.

### Expected behavior

The player should not capture the Home and End keys.

### Browser, and OS with Version.

Firefox 128, W10.

**Comment by Bnyro** [comment]
Imo it's reasonable enough to capture home and end keys, that's been added with https://github.com/TeamPiped/Piped/pull/3664 and should be the expected behavior.

**Comment by AndyRusso** [comment]
I agree with OP. I have lost my position in videos when pressing home key while being deep into reading comments to go back up - it's not intuitive to capture home and end keys at all times in my opinion.
